qt/scientific_interfaces/Inelastic/Analysis/IndirectFitOutputOptionsView.cpp

```diff
@@ -103,6 +115,20 @@ void IndirectFitOutputOptionsView::setEditResultVisible(bool visible) {
   m_outputOptions->pbEditResult->setVisible(visible);
 }
 
+void IndirectFitOutputOptionsView::handleEditResultClicked() {
+  m_editResultsDialog = new IndirectEditResultsDialog(this);
```

Comment:
This causes a memory leak since handleCloseEditResultDialog does not free the memory and clicking on `Edit Result` button keeps on creating this object.

Reply:
Thanks for spotting this. I have now checked that the dialog is definitely being deleted as the destructor is now called when closing the dialog